examples/barchart.rs

```diff
@@ -14,289 +14,358 @@
 //! [examples readme]: https://github.com/ratatui-org/ratatui/blob/main/examples/README.md
 
 use std::{
-    error::Error,
     io,
     time::{Duration, Instant},
 };
 
+use common::Terminal;
```

Comment:
Clarify where this comes from

```suggestion
use self::common::Terminal;
```